pkg/linter/lintBlockedTCPReposnse.go

```diff
@@ -41,11 +41,12 @@ func (lint *blockedTCPResponseLint) lintDescription() string {
 }
 
 func (lint *blockedTCPResponseLint) check() error {
-	for _, nodesConn := range lint.nodesConn {
-		for _, line := range nodesConn.GroupedConnectivity.GroupedLines {
-			tcpRspDisable := line.CommonProperties.Conn.TCPRspDisable
+	for i := range lint.nodesConn {
+		for i2 := range lint.nodesConn[i].GroupedConnectivity.GroupedLines {
+			tcpRspDisable := lint.nodesConn[i].GroupedConnectivity.GroupedLines[i2].CommonProperties.Conn.TCPRspDisable
 			if !tcpRspDisable.IsEmpty() {
-				lint.addFinding(&blockedTCPResponseConn{src: line.Src, dst: line.Dst, tcpRspDisable: tcpRspDisable})
+				lint.addFinding(&blockedTCPResponseConn{src: lint.nodesConn[i].GroupedConnectivity.GroupedLines[i2].Src,
+					dst: lint.nodesConn[i].GroupedConnectivity.GroupedLines[i2].Dst, tcpRspDisable: tcpRspDisable})
```

Comment:
No struct copying took place here before. The map values in `nodesConn` were pointers, and the slice value in `GroupedConnectivity.GroupedLines` were also pointers.

Old code was therefore better (same efficiency, better readability).

Please check all changes in the last commit, and see where pointers are being used (no change needed) and where structs are actually being copied.